src/routing_table.c

```diff
@@ -293,94 +247,82 @@ uint16_t RoutingTB_GetNodeID(uint16_t index)
  * @param None
  * @return None
  ******************************************************************************/
-void RoutingTB_ComputeRoutingTableEntryNB(void)
-{
-    for (uint16_t i = 0; i < MAX_RTB_ENTRY; i++)
-    {
-        if (routing_table[i].mode == CONTAINER)
-        {
+void RoutingTB_ComputeRoutingTableEntryNB(void) {
+    for (uint16_t i = 0; i < MAX_RTB_ENTRY; i++) {
+        if (routing_table[i].mode == CONTAINER) {
             last_container = routing_table[i].id;
         }
-        if (routing_table[i].mode == CLEAR)
-        {
+        if (routing_table[i].mode == CLEAR) {
             last_routing_table_entry = i;
             return;
         }
     }
     // Routing table space is full.
     last_routing_table_entry = MAX_RTB_ENTRY - 1;
 }
+
 /******************************************************************************
  * @brief manage container name increment to never have same alias
  * @param alias to change
  * @param nb to add
  * @return None
  ******************************************************************************/
-static void RoutingTB_AddNumToAlias(char *alias, uint8_t num)
-{
+static void RoutingTB_AddNumToAlias(char *alias, uint8_t num) {
     uint8_t intsize = 1;
-    if (num > 9)
-    {
+    if (num > 9) {
```

Comment:
What is `9` a comment or a macro would be useful.

Reply:
above 9 -> 2 decimal. seem obvious

Reply:
You check here the length of decimal in `num`  this is not the first thing come in mind. Seem obvious for you, but for us as a contributor is not. I think to be more explicit is better.